src/osbuild-manifests/coreos.osbuild.x86_64.mpp.yaml

```diff
@@ -39,11 +39,11 @@ mpp-vars:
   # matches. Unfortunately for FCOS there is no python so we can't
   # really use FCOS as the buildroot so we'll use the host as the
   # buildroot there.
-  buildroot:
-    mpp-if: osname in ['rhcos', 'scos']
-    then: "name:deployed-tree"
-    else:
-      mpp-format-string: '{host_as_buildroot}'
+  buildroot: "name:deployed-tree"
+    # mpp-if: osname in ['rhcos', 'scos', 'fc']
+    # then: "name:deployed-tree"
+    # else:
+    #   mpp-format-string: '{host_as_buildroot}'
```

Comment:
@dustymabe is there a way to condition that to rawhide somehow ? 
I suppose we can't have a different manifest file for rawhide ?

Reply:
We'd have to use an input variable if we wanted that. We could possibly have used ostree-ref but it's actually no longer used so I opened https://github.com/coreos/coreos-assembler/pull/4241